### DISCUSSION TITLE
[Bug] You can't sign in to this app because Google can't confirm that it's safe. (Nov 16, 2022)

### DISCUSSION THREAD

**Discussion by qnblackcat** [qnblackcat/uYouPlus#683] [General]
~~- As the title says. Google just blocked new sign-in on tweaked YouTube apps for the third time. (See the [first time](https://github.com/qnblackcat/uYouPlus/discussions/341) and the [second time](https://github.com/qnblackcat/uYouPlus/discussions/447).~~
~~- No workaround is available.~~
~~- Maybe someone could find a workaround, maybe not. This is just a cat-and-mouse game with Google~~

### ✅ This issue has been solved in uYouPlus [v18.08.1-2.3.1](https://github.com/qnblackcat/uYouPlus/releases/tag/v18.08.1-2.3.1) or higher.

**Comment by EdoardoAb** [comment]
Same problem, I managed to sign in yesterday with my phone. Tried to set it up today on a different Iphone and I cannot get around this issue, I have tried all the known workarounds.

I also tried to disable 2FA on the Google Account, did not help,

**Comment by JellyPeixe** [comment]
Can confirm this issue. Using the latest version of uYouPlus available. Tried logging in on November 16. When using the login shortcut I end up in a sign in loop.

**Reply by themrsnoop** [reply]
Same, on the latest version of uYou but I can use my alt account but not my main for some reason. I have tried changing versions but nothing helps

**Comment by CurssedCoffin** [comment]
same here with ios16.1

**Comment by CurssedCoffin** [comment]
I wonder if i will auto sing out if i log in and google breaks again

**Comment by JioChoi** [comment]
Confirmed on iPad OS 15.6.1.

**Comment by rrroyal** [comment]
confirmed iOS 16.2 (20C5049e) + [uYouPlus v17.43.1-2.1](https://github.com/qnblackcat/uYouPlus/releases/tag/v17.43.1-2.1)

**Comment by Wacypro** [comment]
What do we even do to resolve this?

**Reply by qnblackcat** [reply]
Thats the point. *We don’t know* :/

**Comment by System64exe** [comment]
Same problem, though the only reason that its only happening its due sideloaded through altstore on my end. Only work around i've found to avoid this problem is to have both UYouPlus one from altstore just to download videos, While i'll use this one as example Signulous since im able to sign with it and log in and watch the videos there. 
IOS: 16.2 (20c5049e)
IOS 16.2 (a) (20c7750490e) Security Patch

**Comment by JoeyVers** [comment]
Ahh man, I just wanted to start using uYouPlus from today. Seems I'm two days too late haha.

**Reply by ghost** [reply]
same haha

**Reply by CurssedCoffin** [reply]
same and it happened the night when I want to try third party YouTube app lol
This is the first time I’m trying on ios
Terrible

**Comment by MikeArdor** [comment]
Well, we can still create a new account

**Reply by dnm123k** [reply]
Damn, just realised this
It I’ll use a custom account for my iPad until we get a new update for sign in, Thanks a ton

**Comment by jefferyyongjh** [comment]
Workaround from @AwfulJesse:
What you need to do is type your email and press next on BOTH apps. (Don't put in your password yet) THEN copy+paste the URLs to the short-cut and when the window opens put your password. Then open the uYouplus app and voila.
Credits to @AwfulJesse for finding the workaround!

_Originally posted by @AwfulJesse in
https://github.com/qnblackcat/uYouPlus/discussions/447

**Reply by ghost** [reply]
i think you messed up the link its
[https://github.com/qnblackcat/uYouPlus/discussions/447](https://github.com/qnblackcat/uYouPlus/discussions/447?sort=new#discussioncomment-4182812)

**Reply by jefferyyongjh** [reply]
Made the changes, thanks for informing

**Reply by CurssedCoffin** [reply]
I tried this out and still cannot log in. Here are my steps:
1. open the vanila app, enter mail and click next
2. open the uyou+ app, enter mail and click next
3. copy link from uyou+ app
4. open the shortcut and enter the link
5. copy link from vanila youtube app
6. reurn to the homescreen of ios by swipe up
7. the new login page shows and nothing happens after I enter the password, nor even the "show password" button

Why I'm doing step 6 is because as I stay at vanila youtube app or uyou+ app, the certification of "youtube wants to login with google account" will come in a row even I pressed the continue button.

**Reply by CurssedCoffin** [reply]
Forgive me I just found there is a new shortcut at the Workaround discussion and that works well. I have logged in with steps above.

**Reply by jefferyyongjh** [reply]
No problem, here is a guide for everyone: 
1. Open vanilla app, enter email, click next
2. Copy the link to Notes app
3. Open uYou+, enter email, click next
4. Copy the link
5. Open the shortcut (https://www.icloud.com/shortcuts/cfcc068dfeab4d09a5370fb5126bb58e) , type in the uYou+ link
6. The dialog will disappear for 10 seconds. In the meanwhile, go to your Notes app and copy the link from vanilla app.
7. Once the dialog appears, paste the link from Notes app.
8. A new login page will appear, type in your password and continue.
9. A popup dialog will open and ask to continue in Youtube app, click Open.
10. Click Done at the upper left corner
11. 🎉

**Reply by iulo** [reply]
It work! thanks @jefferyyongjh

**Reply by whalemare** [reply]
@jefferyyongjh please, add next points to guide

0. Disable 2 auth for your account (verification codes just not to be sended to your device, and you unable to go further)  
....
11. Enable 2 auth for your account
12. 🎉

**Reply by rrroyal** [reply]
~~doesn't work for me 😕
iOS 16.2 (20C5049e) + [uYouPlus v17.43.1-2.1](https://github.com/qnblackcat/uYouPlus/releases/tag/v17.43.1-2.1), installed with Sideloadly~~

works after reinstalling 🥳

**Reply by TrigCodes** [reply]
Not sure if Google made any changes on their side again but this does not seem to work for me anymore. When signing into uYou+ and entering my email and hitting next, I get "You can't sign into this app because Google can't confirm that it's safe." instead of asking for my password. As I continue with the steps and enter the vanilla app link into the shortcut prompt, the popup dialogue asks to enter my email instead of asking for my password. Have attempted the above steps multiple times with the same results.

**Reply by a4562248** [reply]
works too, thanks.   iOS 15.7.1  iPhone 13mini

**Reply by Aljeksandr** [reply]
M1 iPad 12.9 2021 1TB not work, with. Shortcut so doesn’t work, sideloadly install and original app from AppStore

**Reply by Aljeksandr** [reply]
On 11 pro max 13.6 all work just login

**Reply by Aljeksandr** [reply]
15.7 on iPad don’t work*

**Reply by Aljeksandr** [reply]
Manual doesn’t work https://imgur.com/a/vVbh2HZ

**Reply by darksaviorx** [reply]
> Manual doesn’t work https://imgur.com/a/vVbh2HZ

It's working here. Read the new instructions.

**Reply by theRiggg** [reply]
Confirmed! It F***** WORKS !!!!@darksaviorx posted the right one ! Enter e-mail then click next on both youtube and uyou, then paste the addresses in shortcut but you need to go back to the shortcuts screen before the 10 seconds mark because if you dont you gonna be stuck in sign in loop.

**Reply by MuhammedKpln** [reply]
Confirming that this works on iOS 16.1.1

**Reply by plooshi** [reply]
confirm

**Reply by cubewcw** [reply]
This totally works! Thanks!

**Reply by agarciacif** [reply]
Working!

**Comment by JoeyVers** [comment]
Nice one man, this works! Thanks a lot!

**Comment by hiddensea** [comment]
This worked for me,Thanks!Sent from my iPhoneOn 19 Nov 2022, at 08:26, jefferyyongjh ***@***.***> wrote:﻿
No problem, here is a guide for everyone:

Open vanilla app, enter email, click next
Copy the link to Notes app
Open uYou+, enter email, click next
Copy the link
Open the shortcut (https://www.icloud.com/shortcuts/cfcc068dfeab4d09a5370fb5126bb58e) , type in the uYou+ link
The dialog will disappear for 10 seconds. In the meanwhile, go to your Notes app and copy the link from vanilla app.
Once the dialog appears, paste the link from Notes app.
A new login page will appear, type in your password and continue.
A popup dialog will open and ask to continue in Youtube app, click Open.
Click Done at the upper left corner
🎉

**Reply by signer64** [reply]
This works

**Reply by KULINININ** [reply]
Thanks, works

**Reply by powerjiayun** [reply]
This works +1

**Reply by PrismSpirit** [reply]
It works! Appreciate it

**Reply by marco20240618** [reply]
works +1.    can't imagine google puts so much effort to block an app that is used by probably only a few thousand people.

**Reply by mytuhuyenninh** [reply]
this work , i have use youtube app link, but copy link to NOTE , not direct to shorcuts 
1 , login 2 app Youtube and uyoutube , not enter pass 
2. copy link youtube > note 
3 copy link uyoutube > shorcuts 
4 go back app NOTE fast and copy link from note ( youtube) > go back shortcuts > it ask link from youtube > paste > it go link like safari, enter password , aks open youtube> click open, wait is done

**Reply by huyle0406** [reply]
Sorry, but this method not work anymore, I input my email and press next in Uyou app and immediately show login error, it not showing the screen where you can input your password.

**Reply by rezevil** [reply]
This method still works, I have just done exactly as the instructions and successfully logged into uYou after 1 hour of sitting around thinking that my phone has a problem, really thank you

**Reply by Luke0722** [reply]
I install cercube plus and this way also works！Thank you！

**Reply by iyuki0430** [reply]
works +1, and thank you!

**Reply by DmzTheo** [reply]
works for me too thx

**Reply by haruhir0219** [reply]
Doesn't work...
I followed exactly as told in the vid and instructions.
I tried two versions I found on this thread.
A: Copy and paste after entering email
B: Copy and paste before entering email


A does prompt me to open youtube after entering password but doesn't open uYouplus app and gets stuck showing rainbow loading animation. 
![ADFD2805-63EC-4792-9AAF-33BA837A4D2E](https://user-images.githubusercontent.com/79664720/205297241-881add49-8ccb-4dfa-bb44-3a8db467143f.png)
![A7EE1B2D-E141-4A07-AF16-63386AE37220](https://user-images.githubusercontent.com/79664720/205297263-45a0e04b-ac15-4eaa-b6ee-e8d3715d6522.png)
Pressing done does returns me to regular YouTube app.

B After entering password it says
“Couldn’t confirm this login was safe. If you think this is a mistake please try again after quitting the app.
![83E31E09-E56C-4492-AA93-3D80922A7ED9](https://user-images.githubusercontent.com/79664720/205297559-a474555f-c01f-4e5a-8102-efd6d63a545a.png)

**Reply by mytuhuyenninh** [reply]
Try solution from me
1. enter login to uYouplus 
2. Enter logoin to youtube 
3. ALL NOT ENTER PASSWORD
4. COPY link from Youtube paste to NOTE " icloud note "
5. Delele youtube app 
6. Copy link from uYouplus to shortcuts go back and copy from note > back to shortcuts it ask link from youtube paste to it > comfirm enter password > open youtube > wait 1 min is done 
7. If You keep youtube app it mean defaul it login to Youtube not uYouplus , that why try copy and delete app Youtube system understand uYouplus = Youtube login is work

**Reply by GrabsOrion** [reply]
@mytuhuyenninh, only yours solution worked for me, many thanks.
For those who get autologin in vanilla YT instead of password prompt - select "Add new account", not plain "Log In".

**Reply by riflerecon** [reply]
worked.

**Reply by MINTY161** [reply]
it's working, thx for u bruuuuuh

**Reply by vinhkosd** [reply]
yeah it worked for me bro thank you very much, My step is
Open vanilla app, enter email, click next

Copy the link and save to somewhere(I use messenger) and Delete vanilla app

Open uYou+, enter email, click next

Copy the link

Open the shortcut (https://www.icloud.com/shortcuts/cfcc068dfeab4d09a5370fb5126bb58e) , type in the uYou+ link i haved copied

The dialog will disappear for 10 seconds. In the meanwhile, I go to messenger app and copy the link from vanilla app i just saved before.

Once the dialog appears, paste the link from Notes app.

A new login page will appear on Safari browser, i typed password and continue.

A popup dialog will open and ask to continue in Youtube app, click Open and It worked.

**Reply by rafarrw** [reply]
> yeah it worked for me bro thank you very much, My step is Open vanilla app, enter email, click next
> 
> Copy the link and save to somewhere(I use messenger) and Delete vanilla app
> 
> Open uYou+, enter email, click next
> 
> Copy the link
> 
> Open the shortcut (https://www.icloud.com/shortcuts/cfcc068dfeab4d09a5370fb5126bb58e) , type in the uYou+ link i haved copied
> 
> The dialog will disappear for 10 seconds. In the meanwhile, I go to messenger app and copy the link from vanilla app i just saved before.
> 
> Once the dialog appears, paste the link from Notes app.
> 
> A new login page will appear on Safari browser, i typed password and continue.
> 
> A popup dialog will open and ask to continue in Youtube app, click Open and It worked.

Thanks, this worked!

**Reply by bushnerd** [reply]
It works.

**Reply by Macleykun** [reply]
Works for me too! Thanks for the shortcut and clear steps!

**Comment by xxxzzz7782** [comment]
所有方法全部是失效，等待大佬更新

**Reply by riflerecon** [reply]
亲测 [hiddensea](https://github.com/hiddensea) 发的那个方法有用 就在楼上 1分钟前刚整好 按照步骤

**Reply by xxxzzz7782** [reply]
感谢🙏，确实有用，再跳转之前，必须删除官方应用

**Reply by xxxzzz7782** [reply]
而且，uyou要用老一点得版本.最新版本，在输入邮箱时，就是提示不安全了

**Reply by riflerecon** [reply]
没删除 都在 最新版uyouplus 都是按照步骤来的

**Comment by a4562248** [comment]
Suddenly I can log in normally after I switch back to this version,  uYouPlus_17.09.1._2.1.ipa. 
The account is displayed when I click login， so I choose the account directly to log in, The account number and password were not entered。

I installed it using altstore, and remove extensions when installed.

**Reply by xxxzzz7782** [reply]
因为你以前用过这个版本登陆信息有所保留，我也遇到这种情况

**Reply by a4562248** [reply]
> 因为你以前用过这个版本登陆信息有所保留，我也遇到这种情况

是的，我之前是一直通过altstore安装旧版本的。后来突然换成其他工具安装新版本的uYOUPlus，就不能登录了。

**Reply by xxxzzz7782** [reply]
我现在也不再用altstore了，现在用牛蛙助手，可以脱离计算机安装应用，就刷刷新签名，没有altstore方便

**Reply by a4562248** [reply]
覆盖安装的时候，牛蛙会干掉之前app的缓存。类似卸载重装了。我就换回altstore了

**Comment by Traitors1991** [comment]
It's 12-29-22 and I cannot get this sign in to work. UYouPlus v2 shortcut literally does not work and shoots back and invalid URL everytime. uYouTube Plus shortcut v1 brings you to a sign in page where as soon as you enter an email, it automatically says you cannot sign in because google can't verify safety. I tried this manually, by substituting the flowname from the UYouPlus youtube sign in page with the flowname from the Appstore Youtube sign in link and then trying to sign in on a private safari and firefox browser. Met with "not safe" error.
I have followed this to a tee. I have done the shortcuts with both emails simply typed into the sign in page, but not hitting next. I have also tried putting the email in both youtube apps and hitting next but without entering any password like the instructions say, but the issue here is that as soon as you put an email into the YouPlus Youtube app, you're not met with an enter password screen like on the Appstore Youtube version, you're met with a "cannot sign in... not safe". I still copied that URL along with the Appstore Youtube URL and im still met with the same issue at the end of the shortcut. You cannot backout of the shortcut at the end anyway because the shortcut opens a safari extension within the shortcut itself instead of straight to the browser so as soon as you backout you have to redo it all over again, which doesn't matter because even if it did open an ACTUAL safari window, the shortcut URL doesn't work anyway.

**Reply by mucookul** [reply]
same issue, anyone please help

**Comment by 12944qwerty** [comment]
If I sideload the updated ipa, will I have to sign in again? It hasn't happened before, but I'm cautious right now.

**Reply by bushnerd** [reply]
Nope, i updates the ipa every week, the login info is reserved.

**Reply by TinTran64** [reply]
> Nope, i updates the ipa every week, the login info is reserved.

@bushnerd Did u experience #360 while updating the IPA?

**Comment by Ronygithu** [comment]
I have a solution about this problem

**Reply by Ronygithu** [reply]
https://github.com/qnblackcat/uYouPlus/discussions/683#discussioncomment-4183063

**Comment by Ronygithu** [comment]
> - As the title says. Google just blocked new sign-in on tweaked YouTube apps for the third time. (See the [first time](https://github.com/qnblackcat/uYouPlus/discussions/341) and the [second time](https://github.com/qnblackcat/uYouPlus/discussions/447).
> 
> - No workaround is available. 
> 
> - Maybe someone could find a workaround, maybe not. This is just a cat-and-mouse game with Google

No problem, here is a guide for everyone:

Open vanilla app, enter email, click next
Copy the link to Notes app
Open uYou+, enter email, click next
Copy the link
Open the shortcut (https://www.icloud.com/shortcuts/cfcc068dfeab4d09a5370fb5126bb58e) , type in the uYou+ link
The dialog will disappear for 10 seconds. In the meanwhile, go to your Notes app and copy the link from vanilla app.
Once the dialog appears, paste the link from Notes app.
A new login page will appear, type in your password and continue.
A popup dialog will open and ask to continue in Youtube app, click Open.
Click Done at the upper left corner
🎉

**Reply by qhgary** [reply]
there‘s no chance to enter password now. It seems google has fixed the bug.

**Reply by getBoolean** [reply]
This doesn't work anymore

**Reply by causingh** [reply]
Same here, 

As soon as you enter an email, it automatically says "you cannot sign in because google can't verify safety", unlike before you can still go to entering password page.

**Reply by mihaigusa** [reply]
+1 it seems as google has patched it 💀

**Reply by Hell4578** [reply]
+1

**Reply by komazhou** [reply]
> there‘s no chance to enter password now. It seems google has fixed the bug.

please update the shortcut to 2.2 from https://github.com/qnblackcat/uYouPlus/discussions/904

**Comment by michaelchany** [comment]
So, this is kinda janky, but I think it could be done on other devices.

1. Install the YT app from the App Store.
2. Install TrollStore
3. Try to install uYou+ IPA with TrollStore.  it will warn you that it detects a non-TrollStore installation but will still give you the option to 'force install' it.
4. I was able to log into my YT account with the uYou+ IPA, whereas before it was blocking me.

Edit: OK i"m not even sure if needing to install it over is necessary, as I just installed uYou+ with TrollStore on another device and had no trouble logging in.  Previously I had been using AltStore.

Edit2: Sorry, I should clarify.  My devices (iPhone SE 2nd gen, iPhone SE 1st gen, and iPhone 7) are on iOS 15.1 and I'm using YT v16.42.3 as the v17 versions just crash.

**Comment by getBoolean** [comment]
Current working solution with the new shortcut is in https://github.com/qnblackcat/uYouPlus/discussions/904

I can confirm it works

**Comment by Anti-God** [comment]
> ~- As the title says. Google just blocked new sign-in on tweaked YouTube apps for the third time. (See the [first time](https://github.com/qnblackcat/uYouPlus/discussions/341) and the [second time](https://github.com/qnblackcat/uYouPlus/discussions/447).~ ~- No workaround is available.~ ~- Maybe someone could find a workaround, maybe not. This is just a cat-and-mouse game with Google~
> 
> ### ✅ This issue has been solved in uYouPlus [v18.08.1-2.3.1](https://github.com/qnblackcat/uYouPlus/releases/tag/v18.08.1-2.3.1) or higher.
What’s up? I have tried to login my gmail into uYouPlus version 18.08.1_.2.3.1 but not succeeded. It didn’t work on iOS 16.3.1. Help Please! 
[@t70438299](https://twitter.com/t70438299)
This is the video: https://twitter.com/i/status/1632570016326643712

**Comment by mixartemev** [comment]
The same.. normal login is not working:(
v18.08.1-2.3.1

**Reply by ghost** [reply]
It would appear that Google is knowing that some accounts are using this app and it's blocking them. I am unable to sign in with one account on this app but I can sign in with another, same app, everything the same two different accounts one can sign in one can't. I'm assuming that Google are blocking one of them with something on their server, the solution is to create a new account and try to sign in again. If you create a new account using the actual app itself, you will be able to sign in and stay signed in. I also recommend using sideloadly instead, as it is able to overwrite previous apps for more easily than the alternatives, especially if you are using a non-paid for developer account, which only gives you seven days for signing. Hope this information helps.

**Reply by mixartemev** [reply]
Yes, new account loged in successfully, but after resigning and reupload app to phone, this account already not logging in:(

**Reply by ghost** [reply]
> Yes, new account loged in successfully, but after resigning and reupload app to phone, this account already not logging in:(

What I found works is to use two different accounts, then I use the new account on the app for say two weeks then for some reason it seems to drop off on Google's end and then I am able to sign in with that blocked account again, but what I also did was because the notifications weren't working on the latest version, I used the latest version to sign in with both accounts so sign in with one account, then click add account then switch between the accounts. then using sideloadly I downgraded the app to the previous version with the notifications that were working now I have two accounts on the app. I am able to sign in with both of them and I get the notifications working and Google are no longer blocking the account, but there does seem to be a time that you have to wait before the block drops off. I'm not sure how long that is but I would say it's at least 10 days.
